src/store.c

```diff
@@ -351,14 +352,18 @@ int pthreads_store_write(zval *object, zval *key, zval *write) {
 				normal refcounting, we'll just never read the reference
 			*/
 			rebuild_object_properties(&threaded->std);
+			
+			pthreads_store_sync(object);
```

Comment:
Producer/Consumer: Producer Thread A writes properties and never reads them. Consumer Thread B reads/removes properties. Table std.properties in Thread A fills up, endlessly. I could limit `pthreads_store_sync` to Volatile objects, that might be a solution you like?